tested/dsl/ast_translator.py

```diff
@@ -333,6 +335,24 @@ def _translate_to_ast(node: ast.Interactive, is_return: bool) -> Statement:
         return _convert_statement(statement_or_expression)
 
 
+def extract_comment(code: str) -> str:
+    """
+    Extract the comment from the code.
+
+    :param code: The code to extract the comment from.
+    :return: The comment if it exists, otherwise an empty string.
+    """
+    tokens = tokenize.tokenize(BytesIO(code.encode("utf-8")).readline)
+    comments = list(
+        map(lambda t: t.string, filter(lambda t: t.type == tokenize.COMMENT, tokens))
+    )
+    if len(comments) == 0:
+        return ""
+    comment = comments[0][1:]
+    assert isinstance(comment, str)
```

Comment:
The `.string` property of a `tokenize.COMMENT` token is guaranteed to be a string according to the documentation, so this extra check is not needed.

Reply:
Resolved in new commit